SwiftCharts/ChartCoordsSpace.swift

```diff
@@ -8,6 +8,33 @@
 
 import UIKit
 
+/**
```

Comment:
This is a helper class that calculates the chart inner frame and generates the axis layers, based on given axis models, (...)